Tools/LambdaTestTool/src/Amazon.Lambda.TestTool/Runtime/LambdaFunctionInfo.cs

```diff
@@ -11,5 +11,15 @@ public class LambdaFunctionInfo
         /// The Lambda function handler string.
         /// </summary>
         public string Handler { get; set; }
+
+        /// <summary>
+        /// http path for the lambda function
+        /// </summary>
+        public string Path { get; set; }
```

Comment:
`Path` could mean a lot in this class's context. What about `HttpResourcePath`?

Reply:
Agreed. Will make the change